examples/kubernetes/nfs-provisioner/nfs-server.yaml

```diff
@@ -1,39 +1,59 @@
+---
 kind: Service
 apiVersion: v1
 metadata:
-  name: nfs-service
+  name: nfs-server
+  labels:
+    app: nfs-server
 spec:
-  clusterIP: 10.0.171.239
+  type: ClusterIP  # use "LoadBalancer" to get a public ip
```

Comment:
does this work in a kind cluster?

Reply:
`ClusterIP` type should always work, anyway it's better than a fixed IP config.